addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -154,12 +152,12 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		if len(errs) > 0 {
 			return ctrl.Result{}, fmt.Errorf("few failures occured while creating VolumeReplicationClasses: %v", errs)
 		}
-	}
 
-	klog.Infof("labeling rbd storageclasses")
-	errs := r.labelRBDStorageClasses(ctx, mirrorPeer, scr.Namespace, clusterFSIDs)
-	if len(errs) > 0 {
-		return ctrl.Result{}, fmt.Errorf("few failures occured while labeling RBD StorageClasses: %v", errs)
+		klog.Infof("labeling rbd storageclasses")
```

Comment:
@raghavendra-talur Do we have requirement to label the storage classes?

Reply:
This can be skipped for now.